code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -273,4 +274,8 @@ Pipelines + Other Objects -> Pipe network
 
 
 /obj/machinery/atmospherics/proc/can_crawl_through()
-	return 1
\ No newline at end of file
+	return 1
+
+/obj/machinery/atmospherics/proc/returnPipenets()
+	return

```

Comment:
You sure that XORing null with a list works the same as XORing an empty list with a list?